src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/data/WorkReadinessStatusAndDetails.kt

```diff
@@ -5,44 +5,4 @@ import java.time.LocalDate
 data class WorkReadinessStatusAndDetails(
```

Comment:
Would be good to rename this to just `WorkReadinessStatus` as we've removed the `details`